scripts/azure-linux_mac.yml

```diff
@@ -263,15 +238,4 @@ steps:
          cat $f
       done;
   condition: failed() # only run this job if the build step failed
-  displayName: "Print log files (failed build only)"
-
-- bash: |
-    set -e pipefail
-    python -m pip install --upgrade pip virtualenv
-
-    # Build the documentation (this does not deploy to RTD).
-    pushd $BUILD_REPOSITORY_LOCALPATH/doc;
-    ./local-build.sh;
-    popd;
-  condition: eq(variables['Agent.OS'], 'Linux')
-  displayName: 'Build docs (no deployment)'
+  displayName: "Print log files (failed build only)"
```

Comment:
Several of the files are indicating missing newline at end of file, don't know if that will cause problems anywhere.

Reply:
Addressed, thanks!